packages/forms-web-app/src/pages/projects/register/agent/router.js

```diff
@@ -181,11 +178,15 @@ registerAgentRouter.post(
 
 registerAgentRouter.get(
 	registerAgentOrgNameURL,
+	addCommonTranslationsMiddleware,
```

Comment:
Could the middleware that's being used on multiple routes be added at router level?

`registerAgentRouter.use(addCommonTranslationsMiddleware, addRegisterTranslationsMiddleware)`

Reply:
good call - have tidied this up across the register to have your say routes